code/controllers/subsystems/responseteam.dm

```diff
@@ -2,123 +2,96 @@
 
 /datum/controller/subsystem/responseteam
 	name = "Response Team"
-	wait = 1800 //Fire only every 3 minutes - Not more often needed for now
+	flags = SS_NO_FIRE
 
-	var/progression_chance = 0
-	var/percentage_antagonists = 0
-	var/percentage_dead = 0
-	var/can_call_ert = TRUE
-	var/ert_type = "NT-ERT" //what ert type will be deployed
-	var/send_emergency_team = 0
 	var/ert_count = 0
+	var/send_emergency_team = FALSE
+	var/can_call_ert = TRUE
 
-	var/pg_green = 0
-	var/pg_yellow = 0
-	var/pg_red = 0
-	var/pg_delta = 0
+	var/list/datum/responseteam/available_teams = list()
+	var/datum/responseteam/picked_team
+	var/list/datum/ghostspawner/human/ert/sent_teams = list()
 
 /datum/controller/subsystem/responseteam/Recover()
-	progression_chance = SSresponseteam.progression_chance
-	can_call_ert = SSresponseteam.can_call_ert
-	ert_type = SSresponseteam.ert_type
 	send_emergency_team = SSresponseteam.send_emergency_team
-	pg_green = SSresponseteam.pg_green
-	pg_yellow = SSresponseteam.pg_yellow
-	pg_red = SSresponseteam.pg_red
-	pg_delta = SSresponseteam.pg_delta
 
 /datum/controller/subsystem/responseteam/New()
 	NEW_SS_GLOBAL(SSresponseteam)
 	feedback_set("responseteam_count",0)
 
+/datum/controller/subsystem/responseteam/Initialize(start_timeofday)
+	. = ..()
+	var/list/all_teams = subtypesof(/datum/responseteam)
+	if(!all_teams)
+		log_debug("No response teams found!")
+		return
+	else
```

Comment:
this else is unnecessary.

Reply:
Shit, how could you do this to me? I trusted you, Skull. How dare you ruin my life like this. How dare you shit on my code like this. This else was the product of 3 hours of consecutive thinking. And you just came in here like this. Why would you do this, Skull?